Misc/NEWS.d/next/Library/2025-03-09-11-01-00.gh-issue-83461.auwd13.rst

```diff
@@ -0,0 +1,3 @@
+:meth:`datetime.date.strptime`, :meth:`datetime.datetime.strptime`,
+:meth:`datetime.time.strptime` and :meth:`time.strptime` now only accept ASCII
+digits.
```

Comment:
Now that I read it, it's as if we only accept ASCII digits as inputs but maybe I'm biased. So maybe "Digits passed to [...] must now be ASCII.".